solc/CommandLineInterface.cpp

```diff
@@ -610,6 +614,23 @@ Allowed options)",
 
 bool CommandLineInterface::processInput()
 {
+	if (m_args.count(g_argStandardJSON))
+	{
+		string input;
+		while (!cin.eof())
+		{
+			string tmp;
+			getline(cin, tmp);
+			input.append(tmp + "\n");
+		}
+//		string input = R"({ "language": "Solidity", "sources": { "mortal": { "keccak256": "0x234...", "content": "contract owned { address owner; } contract mortal is owned { function kill() { if (msg.sender == owner) selfdestruct(owner); } }" } }, "settings": { "remappings": [ ":g/dir" ], "optimizer": { "enabled": true, "runs": 500 }, "metadata": { "useLiteralContent": true }, "libraries": { "myFile.sol": { "MyLib": "0x123123" } }, "outputSelection": { "*": { "*": [ "metadata", "evm.bytecode" ], "": [ "ast", "why3" ] } } } })";
+		StandardCompiler compiler;
+		string output = compiler.compile(input);
+		cout << "Standard Input JSON: " << input << endl << endl;
+		cout << "Standard Output JSON: " << output << endl;
+		return false;
```

Comment:
The CLI is split into `processInput` and `actOnInput`. I'm not sure if this division makes sense here, but I think we should only run the compile step in the `actOnInput` function.